packages/visualizations/src/components/MapPoi/MapRender.svelte

```diff
@@ -31,9 +40,29 @@
 </script>
 
 <figure class="map-card maps-container" style={cssVarStyles}>
-    <div class="main">
+    {#if title || subtitle}
+        <figcaption>
+            {#if title}
+                <h3>
+                    {title}
+                </h3>
+            {/if}
+            {#if subtitle}
+                <p>
+                    {subtitle}
+                </p>
+            {/if}
+        </figcaption>
```

Comment:
I totally agree with this point, it would be a nice opportunity to create a new component to display that. Let's create a ticket and do it in a separate PR to not mix too much the updates !

Reply:
will do later, see https://app.shortcut.com/opendatasoft/story/43434